src/test/java/uk/gov/digital/ho/pttg/testrunner/IpsClientTest.java

```diff
@@ -74,6 +76,28 @@ public void sendFinancialStatusRequest_anyRequest_includeBasicAuthHeader() {
         assertThat(headers.get("Authorization").get(0)).isEqualTo("Basic " + Base64.getEncoder().encodeToString(SOME_BASIC_AUTH.getBytes()));
     }
 
+    @Test
+    public void sendFinancialStatusRequest_anyRequest_includeCorrelationId() {
+        ipsClient.sendFinancialStatusRequest(ANY_REQUEST);
+
+        then(mockRestTemplate).should()
+                              .exchange(eq(SOME_IPS_ENDPOINT), eq(POST), httpEntityCaptor.capture(), eq(Void.class));
+        HttpHeaders headers = httpEntityCaptor.getValue().getHeaders();
+        String correlationId = headers.get("x-correlation-id").get(0);
```

Comment:
Why not use a specific UUID so you can confirm that it is transformed both ways, e.g. UUID.fromString("aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee");